ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/policy-maps.md

```diff
@@ -69,20 +69,30 @@ policy-map type pbr PM_PBR_BREAKOUT
 
 ##### PM_REPLICATION_LD
 
-| class | Set | Value |
-| ----- | --- | ----- |
-| CM_REPLICATION_LD | dscp | af11 |
-| CM_REPLICATION_LD | traffic_class | 2 |
-| CM_REPLICATION_LD | drop_precedence | 1 |
-| CM_REPLICATION_LD_2 | dscp | af11 |
-| CM_REPLICATION_LD_2 | traffic_class | 2 |
+| class | Set | Value | Police |
```

Comment:
Lets change this to
```suggestion
| Class | DSCP | Traffic Class | Drop Precedenc | Police |
```

Reply:
fixed in https://github.com/aristanetworks/avd/pull/3620/commits/5c80e23ea5dbd17b4c843fd3a289ed1998209330